packages/block-library/src/navigation/edit/accessible-description.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { cloneElement } from '@wordpress/element';
+import { VisuallyHidden } from '@wordpress/components';
+
+export default function AccessibleDescription( {
+	id,
+	content,
+	isPlaceholder,
+	children,
+} ) {
+	if ( ! children ) {
+		return null;
+	}
+
+	if ( isPlaceholder ) {
+		return children;
+	}
```

Comment:
I don't like that this component has to know about the concept of a placeholder.